app/components/post_body/post_body.test.js

```diff
@@ -122,4 +123,13 @@ describe('PostBody', () => {
         instance.measurePost(event);
         expect(wrapper.state('isLongPost')).toEqual(false);
     });
+
+    test('should return system message as post body', () => {
+        const mockSystemMessageComponent = jest.fn(() => ('A rendered system message component.'));
+        SystemMessageHelpers.renderSystemMessage = mockSystemMessageComponent;
```

Comment:
Instead of modifying `SystemMessageHelpers` directly, this should be mocked by adding a call to `jest.mock` below the imports at the top of this file like
```javascript
jest.mock('./system_message_helpers');
```

Reply:
Of course! Duh.

Done.